includes/functions.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+	// Adding custom fields (ACF) to rest api responses
+		function slider_custom_rest_api () {
+			register_rest_field(
+				'verticalsliders',
+				'test-name',
```

Comment:
I realize this is a test name, but JSON field names should only be lowercase and only contain underscores. The field name `test-name` would not pass JSON validation.